cmd/create/cluster/cmd.go

```diff
@@ -1128,6 +1128,7 @@ func run(cmd *cobra.Command, _ []string) {
 					)
 				} else {
 					r.Reporter.Errorf("A billing account is required for Hosted Control Plane clusters. %s", listBillingAccountMessage)
+					os.Exit(1)
```

Comment:
Could you refactor to capture this validation within a helper function and add a small unit test around just the function?

Reply:
Sure, I created helper function for providing billing account and added unit tests for it.